Documentation/dev-guide/apispec/swagger/rpc.swagger.json

```diff
@@ -615,17 +615,7 @@
           "Lease"
         ],
         "summary": "LeaseLeases lists all existing leases.",
-        "operationId": "LeaseLeases",
-        "parameters": [
```

Comment:
The parameters blocks such as this getting deleted in the auto-generated file doesn't seem right. Maybe a problem in the way "additional-bindings" work?

Reply:
Yeah something is amiss, it works but not consistently. Possible upstream bug, but not sure.